makeflow/src/makeflow_gc.c

```diff
@@ -147,16 +147,14 @@ int makeflow_clean_file( struct dag *d, struct batch_queue *queue, struct dag_fi
 		makeflow_log_file_state_change(d, f, DAG_FILE_STATE_DELETE);
 
 		if(!silent)
-			debug(D_NOTICE, "Makeflow: Deleted path %s\n", f->filename);
+			debug(D_MAKEFLOW_RUN, "Makeflow: Deleted path %s\n", f->filename);
```

Comment:
I understand removing the others, but this is the only location where the removal of a file is reported to the user. In older versions of Makeflow we had a comment such as "makeflow deleted file ..." that always printed when a file was cleaned. @btovar Should clean reported removed files if not in debug?